packages/demand-bloc-payment/src/types.ts

```diff
@@ -17,7 +18,12 @@ export type Payer = {
   last_name: string
 }
 
-type TokenizePayload = string[]
+type TokenizePayload = {
+  nonce: string
+  details?: HostedFieldsAccountDetails
```

Comment:
`TokenizePayload` is common type for all providers, so we cannot use Braintree specific type for one of properties.
Let's use any for now and fix later